esx_service/vmodl/vmodl_test.py

```diff
@@ -497,6 +489,29 @@ def test_add_privilege(self):
         # Verify the default datastore
         self.assertEqual(result[0].default_datastore, self.datastore)
 
+    def test_add_privilege_without_volume_max_size(self):
+        # Create a tenant
+        tenant = self.tenantMgr.CreateTenant(name=TENANT_NAME, description=TENANT_DESC)
+
+        # Create a privilege without volume_max_size
+        privilege = vim.vcs.storage.DatastoreAccessPrivilege()
+        privilege.datastore =  self.datastore
+        privilege.allow_create = True
+        privilege.volume_total_size = 1024
+
+        # Add privilege to the tenant
+        self.tenantMgr.AddPrivilege(tenant, privilege)
+
+        # Get the privilege
+        result = self.tenantMgr.GetTenants(name=TENANT_NAME)
+        self.assertTrue(result)
+        p = result[0].privileges
+        self.assertTrue(p)
+
+        # Verify that the volume_max_size was set to the same value as volume_total_size
+        self.assertEqual(p[0].volume_total_size, 1024)
```

Comment:
Same as comments above. I don't understand the logic that set volume_max_size to the same value as volume_total_size.